episodes/infrastructure.md

```diff
@@ -214,12 +214,15 @@ it is safest to replace it with another value within quotation marks.
 
 ### Practice with `config.yaml` (5 minutes)
 
-Complete the configuration of your lesson by adjusting the following fields in `config.yaml`:
+Complete the configuration of your lesson by adjusting the following fields in `config.yaml`. 
+Remember that FIXME values with quotes should be replaced with another value still in quotes and 
+FIXMEs without quotes should be replaced by values without quotes.
 
-- `email`: add an email address people can contact with questions about the lesson/project.
+- `contact`: add an email address people can contact with questions about the lesson/project.
 - `created`: the date the lesson was created (today's date) in YYYY-MM-DD format.
 - `keywords`: a (short) list of keywords for the lesson,
-   which can help people find your lesson when searching for resources online.
+   which can help people find your lesson when searching for resources online. At a minimum,
+  include 'lesson', ‘carpentries-incubator’, your lesson's life-cycle stage, and your lesson's (human) language.
```

Comment:
```suggestion
  include 'lesson', 'carpentries-incubator', your lesson's life-cycle stage (one of 'pre-alpha', 'alpha', 'beta', 'stable'), and your lesson's (human) language. For example, 'lesson, carpentries-incubator, pre-alpha, Spanish'
```

Reply:
Should we use "carpentries-incubator" or "Carpentries Incubator"? I can see why one word would be better but in terms of findability, what is better to use here?

Reply:
I like the suggestion to include an example. 

Every trainee's lesson should be at pre-alpha, and we have not explained the life cycle at this point in the curriculum, so it may also be clearer to specify that the life cycle stage to include in the keywords list should be "pre-alpha".

Reply:
Thanks both! I've removed carpentries-incubator, included pre-alpha, and changed "Spanish" to "español".